strawberry/django/views.py

```diff
@@ -177,14 +185,28 @@ def create_response(
 
         return response
 
+    async def create_multipart_response(
+        self,
+        request: HttpRequest,
+        stream: Callable[[], AsyncIterator[Any]],
+        sub_response: HttpResponse,
```

Comment:
Ditto here about `sub_response` (which applies basically to all `create_multipart_response`